service-worker/worker/src/e2e/simple/protractor.config.js

```diff
@@ -0,0 +1,27 @@
+exports.config = {
```

Comment:
Could we move this file to service-worker/worker? Or is there a reason it needs to be this deep (i.e. other end to end tests require significantly different configuration)?